src/main/python/pybuilder/plugins/python/distutils_plugin.py

```diff
@@ -114,7 +114,7 @@ def as_str(value):
 @init
 def initialize_distutils_plugin(project):
     project.plugin_depends_on("pypandoc", "~=1.4")
-    project.plugin_depends_on("setuptools", ">=38.6.0")
+    project.plugin_depends_on("setuptools", "==62.1.0")
```

Comment:
I would suggest to require `<62.2.0` instead of `==62.1.0`, in order to minimize the chance of conflicts during dependency resolution.

Reply:
@lormico amended, commit cbd7901b6bffe8e4d7e4eb435a4c39f2b8880179.